src/helpers/geoChart/data.ts

```diff
@@ -64,6 +65,7 @@ export function getGeoData(
 
     if (segmentIndex !== undefined) {
         geoData[SEGMENT].data = getAttributeData(attributeHeaderItems, segmentIndex);
+        geoData[SEGMENT].uris = getAttributeUris(attributeHeaderItems, segmentIndex);
```

Comment:
do you want to merge these 2 lines into 1?

Reply:
Do you mean?
```
geoData[SEGMENT] = {
    ...geoData[SEGMENT],
    data: getAttributeData(...), 
    uris: getAttributeUris(...)
}
```
If so, I would prefer the first style

Reply:
I meant: we will create a new functon to get all data and uris for segment. 
Use only 1 loop to save time

Reply:
Fixed. Thanks